### DISCUSSION TITLE
Please update pom.xml to use release 2.1.0

### DISCUSSION THREAD

**Discussion by rbotzer** [aerospike/spring-data-aerospike#516] [Team Posts]
I see that the [pom.xml is referring to the previous 2.0.1 release](https://github.com/aerospike-community/spring-data-aerospike/blob/master/pom.xml#L7-L14). I think it should be modified before we move release 2.1.0 from its draft state, no?

Also, the README refers to 2.0.1 rather than 2.1.0.

One other minor question - Considering that on the Aerospike side Peter Milne, Jeff Boone and Michael Zhang have all left the company and no longer active contributors, should the developers section be modified to reflect that? What is customary?  I can edit and replace those with Joe Martin and Brian Nichols later - not something to hold up the release, I'm just curious.

**Comment by rbotzer** [comment]
@aerospike-community/team-playtika I'm not sure if you saw this question, so let me know if the team chat is working.

**Comment by tdanylchuk** [comment]
summoning @Aloren

**Comment by Aloren** [comment]
Hi all.
> I think it should be modified before we move release 2.1.0 from its draft state
> the README refers to 2.0.1 rather than 2.1.0.

Will fix right now.

> the developers section be modified to reflect that?

I don't know what is the correct procedure. We can update the developers section the way you want it to be.

**Comment by Aloren** [comment]
Version updated via https://github.com/aerospike-community/spring-data-aerospike/commit/5450190a9eee41f641d52ba00b3b908b7a84a2d4

**Comment by rbotzer** [comment]
The 2.1.0 jar is up in Maven central repository: https://search.maven.org/search?q=a:spring-data-aerospike

I published the release in the GitHub repository. I hope that's okay.
